panel/widgets/input.py

```diff
@@ -129,6 +129,25 @@ class ColorPicker(Widget):
     _rename = {'value': 'color', 'name': 'title'}
 
 
+class Spinner(Widget):
+
+    low = param.Number(default=None, doc="""
```

Comment:
Let's make these consistent with other similar widgets and call them `start` and `end`.